model/src/main/kotlin/ScannerRun.kt

```diff
@@ -135,6 +135,20 @@ data class ScannerRun(
             }
         }
 
+        scanResults.getDuplicates { it.provenance to it.scanner }.keys.also { duplicates ->
```

Comment:
I missed that existing code is already using `let` instead of `also`. Mind adding a preceding commit in this PR that first changes existing `let` to `also`, with the mentioned rationale that the return value is unneeded.